getting-started.adoc

```diff
@@ -1,3 +1,123 @@
 = Getting Started with Bisq
+:toc: left
+:sectanchors:
+:btc_deposit: 0.01
```

Comment:
Please use dashes vs. underscores in custom Asciidoc attributes like these. This follows conventions with Asciidoc's own built-in attributes (like `figure-caption` below).